reps/2023-05-10-actors-batch-remote-api.md

```diff
@@ -0,0 +1,204 @@
+## Summary
+### General Motivation
+**Core Motivation**:
+1. Improve the performance of batch calling ActorTask.
+2. Implement Ray's native collective communication library through this interface.
```

Comment:
We should split this REP into 2: 
1. batch remote in ray core REP
2. introducing RAY_NATIVE mode in ray collective lib REP

Reply:
Yes, This REP just adds Batch Rmote API.